### PR TITLE
Add Libacvp Package

### PR DESCRIPTION
**Pull request by niontive**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Add Libacvp package. This is needed for FIPS validation.

###### Change Log  

- Add "libacvp" and "libacvp-app" package

###### Does this affect the toolchain?  

No

###### Associated issues  

None

###### Links to CVEs  
None

###### Test Methodology

Local build

### CODE REVIEW

**Review by danmihai1 — APPROVED** [review]

File: SPECS/libacvp/libacvp.spec [hunk 1]
```diff
@@ -0,0 +1,61 @@
+Summary:        A library that implements the client-side of the ACVP protocol
+Name:           libacvp
+Version:        1.2.0
+Release:        1%{?dist}
+License:        Apache License 2.0
```
Comment (PawelWMS) [inline SPECS/libacvp/libacvp.spec:5]: Please use the short name from [Fedora's list](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#License_of_Fedora_SPEC_Files) as per our spec guidelines. In this case it's probably "ASL 2.0".

File: SPECS/libacvp/libacvp.spec [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+%{_datadir}/README.md
+%{_libdir}/libacvp.a
+%{_includedir}/acvp/*
+
+%files app
+%{_bindir}/acvp_app
+
+%changelog
+* Mon Feb 08 2021 Nicolas Ontiveros <niontive@microsoft.com> - 1.2.0-1
+- First version. License verified.
```
Comment (PawelWMS) [inline SPECS/libacvp/libacvp.spec:61]: Please use the phrase "Original version for CBL-Mariner."

Thank you for checking the license!

File: cgmanifest.json [hunk 3]
```diff
@@ -2190,6 +2190,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "libacvp",
```
Comment (PawelWMS) [inline cgmanifest.json:2197]: Please also update the licenses map file under `SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`. I know, I know, we need some automation for all of these requirements...;)

**Review by PawelWMS — CHANGES_REQUESTED** [review]

File: cgmanifest.json (hunk 3 above)
Comment (niontive) [reply cgmanifest.json:2197]: Would this go under the Microsoft MIT license?

Comment (PawelWMS) [reply cgmanifest.json:2197]: Yup, that's correct.

File: cgmanifest.json [hunk 4]
```diff
@@ -2190,6 +2190,16 @@
...
+          "version": "1.2.0",
+          "downloadUrl": "https://github.com/cisco/libacvp/archive/v1.2.0.tar.gz"
```
Comment (PawelWMS) [inline cgmanifest.json:2199]: > "downloadUrl": "https://github.com/cisco/libacvp/archive/v1.2.0.tar.gz" [](start = 9, length = 72)

I've noticed that the [manifest check](https://github.com/microsoft/CBL-Mariner/pull/607/checks?check_run_id=1863788961) is also acting up for some reason. Could you take a look?

Comment (niontive) [reply cgmanifest.json:2199]: Manifest check expects the downloadUrl to be:
`https://github.com/cisco/libacvp/archive/libacvp-1.2.0.tar.gz`

However, the URL I have is correct. Several other packages have this issue (ex: rapidjson), so I won't change this.

Comment (thcrain-msft) [reply cgmanifest.json:2199]: https://github.com/microsoft/CBL-Mariner/pull/616

This PR should fix that- the manifest check just needs a small change to handle the escaped name/version macros in your `#Source0` line.

Comment (PawelWMS) [reply cgmanifest.json:2199]: Loving all of this.:)

---
In reply to: [573258286](https://github.com/microsoft/CBL-Mariner/pull/607#discussion_r573258286) [](ancestors = 573258286)

**Review by PawelWMS — APPROVED** [review]
:shipit:

**Pull request merged**